src/main/pileup.js

```diff
@@ -79,13 +79,13 @@ function create(elOrId: string|Element, params: PileupParams): Pileup {
                          initialRange={params.range} />, el);
   return {
     setRange(range: GenomeRange) {
-      if (reactElement == null) {
+      if (reactElement === null) {
```

Comment:
It's OK here (since `reactElement` is explicitly set to `null` below), but I do want to call out that this change isn't a no-op. `x == null` will be true if `x` is either `null` or `undefined`, but `x === null` will only be true if it's null.